inlang/source-code/sdk/src/test-utilities/createMessage.ts

```diff
@@ -1,6 +1,12 @@
-import type { Message, Pattern } from "@inlang/message"
+import type * as LegacyFormat from "@inlang/message"
 
-export const createMessage = (id: string, patterns: Record<string, Pattern | string>) =>
+/**
+ * @deprecated Uses Legacy Message Format
```

Comment:
@jldec here we use the LegacyFormat - we deprecate this function to allow fink / sherlock etc to adopt the new (to be defined functions)

Reply:
ok - I'm a bit worried about deprecation noise in our build logs - only use if really necessary